mozilla-release/mobile/android/base/java/org/mozilla/gecko/widget/IconTabWidget.java

```diff
@@ -17,9 +18,60 @@
 import android.widget.TextView;
 
 public class IconTabWidget extends TabWidget {
+
+    /* Cliqz Start */
+    private static final int[] STATE_PRIVATE_MODE = { R.attr.state_private };
+
+    private boolean isPrivate;
+
+    private static final int[] LIGHT_THEME = { R.attr.light_theme };
+    private static final int[] LIGHT_THEME_PRIVATE_MODE = { R.attr.light_theme, R.attr.state_private };
+    private boolean isLightTheme;
```

Comment:
`mIsLightTheme`. Are we planning to add a boolean for every other theme we implement or are we using numeric coding?